### DISCUSSION TITLE
ConnectError

### DISCUSSION THREAD

**Discussion by BOURBON0** [acheong08/EdgeGPT#415] [Q&A]
I can only have one conversation, and the second time I ask, I will report an error:
![2023-05-17 214022](https://github.com/acheong08/EdgeGPT/assets/54803330/62d9ac36-0fde-45f6-811f-00b7e499d76e)

**Comment by RccccR** [comment]
metoo!!!

**Reply by RccccR** [reply]
I found a solution!!! The reason is that the core issue is mainly caused by urllib3. The latest urllib3 has stricter SSL verification, so the following solution is used to solve it：
Downgrading urllib3 to version 1.26.15 is a perfect solution！！！

**Reply by RccccR** [reply]
but now,sameQAQ
